tests/functional/Robo/Tasks/CopyToDocker.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Test\Functional\Robo\Tasks;
+
+use Robo\Common\ExecOneCommand;
+use Robo\Contract\CommandInterface;
+use Robo\Contract\TaskInterface;
+use Robo\Result;
+use Robo\Task\BaseTask;
+
+/**
+ * Copy files to Docker environment
+ */
+class CopyToDocker extends BaseTask implements CommandInterface, TaskInterface
+{
+    use ExecOneCommand;
+
+    /**
+     * Container name
+     *
+     * @var string
+     */
+    protected $container;
+
+    /**
+     * Path to file on the Docker environment
+     *
+     * @var string
+     */
+    protected $source;
+
+    /**
+     * Path to file on the local machine
+     *
+     * @var string
+     */
+    protected $destination;
+
+    /**
+     * @param string $source
+     * @param string $destination
+     * @param string $container
+     */
+    public function __construct(string $source, string $destination, string $container)
+    {
+        $this->container = $container;
+        $this->source = $source;
+        $this->destination = $destination;
+    }
```

Comment:
Not a blocker really, but Robo convention is usually a more fluent syntax. Something along the lines of:

```php
$this->copyToDocker($src)
    ->container($containerName)
    ->destination($somePath)
    ->run()
```

This structure of putting everything in the constructor works fine, but is definitely different that the style I've seen elsewhere

Reply:
Or looking at the approach we've done else where, starting with the `$container` in the constructor and `source()` and `destination()` as methods.